Src/IronPythonTest/AssemblyInfo.cs

```diff
@@ -24,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("IronPythonTest")]
-[assembly: AssemblyCopyright("� Microsoft Corporation.  All rights reserved.")]
+[assembly: AssemblyCopyright("� Microsoft Corporation.  All rights reserved.")]
```

Comment:
Copyright symbol is broken. Should probably read IronPython Contributors anyway...

Reply:
Hmmm, it looks fine here, what is broken about it?

Reply:
Shows up as a diamond with a question mark over here.